### PR TITLE
refactor(kit): `InputSlider` use `InputNumber` inside

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the new behavior?
`InputSlider` uses `InputNumber` inside itself.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/kit/components/input-number/input-number.component.ts [hunk 1]
```diff
@@ -281,6 +269,26 @@ export class TuiInputNumberComponent
         this.updatePressed(pressed);
     }
 
+    getFormattedValue(value: number): string {
```
Comment (waterplea) [inline projects/kit/components/input-number/input-number.component.ts:272]: Private?

File: projects/kit/components/input-number/input-number.template.html [hunk 2]
```diff
@@ -21,4 +22,13 @@
 >
     <ng-content></ng-content>
     <ng-content select="input" ngProjectAs="input"></ng-content>
+    <div
```
Comment (waterplea) [inline projects/kit/components/input-number/input-number.template.html:25]: Clever!

File: projects/kit/components/input-slider/input-slider.template.html [hunk 3]
```diff
@@ -1,78 +1,38 @@
...
+    [tuiTextfieldCustomContent]="controller.customContent || deprecatedSecondary"
+    [tuiTextfieldSize]="computedSize"
+    [tuiTextfieldLabelOutside]="computedSize !== 'l'"
     [readOnly]="readOnly"
-    [disabled]="disabled"
-    [focused]="computedFocused"
-    [hovered]="computedHovered"
-    [invalid]="computedInvalid"
+    [pseudoFocused]="computedFocused"
+    [pseudoHovered]="pseudoHovered"
```
Comment (waterplea) [inline projects/kit/components/input-slider/input-slider.template.html:13]: Please use `computed` values for now, they are deprecated but still needed until 3.0 to preserve possible custom appearances. Let's just replace them all together later.

File: projects/kit/components/input-slider/input-slider.component.ts [hunk 4]
```diff
@@ -133,149 +178,133 @@ export class TuiInputSliderComponent
...
+    get computedValueContent(): PolymorpheusContent<TuiContextWithImplicit<number>> {
+        return this.minLabel || this.maxLabel
+            ? legacyMinMaxLabel(this)
+            : this.valueContent;
     }
 
-    get inputValue(): string {
-        return this.focusableElement ? this.focusableElement.nativeElement.value : '';
+    @tuiPure
+    computeContext(value: number): TuiContextWithImplicit<number> {
```
Comment (waterplea) [inline projects/kit/components/input-slider/input-slider.component.ts:220]: This is unnecessary, just write inline object in template, Angular optimizes it.

**Review by waterplea — APPROVED** [review]

File: projects/kit/components/input-slider/input-slider.component.ts (hunk 4 above)
Comment (nsbarsukov) [reply projects/kit/components/input-slider/input-slider.component.ts]: fixed

File: projects/kit/components/input-slider/input-slider.template.html (hunk 3 above)
Comment (nsbarsukov) [reply projects/kit/components/input-slider/input-slider.template.html]: fixed for `InputSlider` and for `InputNumber`

File: projects/kit/components/input-number/input-number.component.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/kit/components/input-number/input-number.component.ts:272]: This method is required also inside `InputSlider`

**Review by vladimirpotekhin — APPROVED** [review]
🔥

**Pull request merged**